requirements/nightly.in

```diff
@@ -0,0 +1,3 @@
+# Generated by 'tox -e deps', DO NOT EDIT MANUALLY!'
+https://github.com/scipp/scipp/releases/download/nightly/scipp-nightly-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
```

Comment:
`cp38-cp38`: has something gone wrong in the name here?

Reply:
This is how it's supposed to be. See here: https://pypi.org/project/scipp/#files
I don't know why it's repeated.